kornia/feature/responses.py

```diff
@@ -0,0 +1,313 @@
+from typing import Tuple, Union
+
+import torch
+import torch.nn as nn
+
+from kornia.filters import spatial_gradient, gaussian_blur2d
+
+
+def harris_response(input: torch.Tensor,
+                    k: Union[torch.Tensor, float] = 0.04,
+                    grads_mode: str = 'sobel',
+                    sigmas: Union[torch.Tensor, None] = None) -> torch.Tensor:
```

Comment:
recently learnt that in this case we should use Optional. `Optional[torch.Tensor] = None`